.gitignore

```diff
@@ -89,10 +88,9 @@ tags
 .vscode/*
 .history
 openshift.local.clusterup/*
-build/_output
```

Comment:
I don't think the changes to this file are necessary. We want to ignore the files/dirs you have removed and the only addition is the Jetbrains comment.

Reply:
k, agree. This's repeated in [L4](https://github.com/infinispan/infinispan-operator/blob/main/.gitignore#L4), hence the removal.